config/main.py

```diff
@@ -686,7 +686,7 @@ def _stop_services():
         pass
 
     click.echo("Stopping SONiC target ...")
-    clicommon.run_command("sudo systemctl stop sonic.target")
+    clicommon.run_command("sudo systemctl stop sonic.target --job-mode replace-irreversibly")
```

Comment:
Can we add this option also for systemctl restart sonic.target case at line#709

Reply:
Added !